packages/utils/parcelforvscode/schemas/package-targets.schema.json

```diff
@@ -0,0 +1,167 @@
+{
+  "title": "JSON schema for .parcelrc files",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "$defs": {
+    "engines": {
+      "type": "object",
+      "properties": {
+        "browsers": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            {
+              "type": "string"
+            }
+          ]
+        }
+      },
+      "additionalProperties": {
+        "type": "string"
+      }
+    },
+    "target": {
+      "type": "object",
+      "properties": {
+        "context": {
+          "description": "The context property defines what type of environment to build for. This tells Parcel what environment-specific APIs are available, e.g. the DOM, Node filesystem APIs.",
```

Comment:
Some of this JSON duplicates what https://github.com/parcel-bundler/parcel/blob/9d6da2c/packages/core/core/src/TargetDescriptor.schema.js does, but descriptions seem novel.

I think if you add descriptions into TargetDescriptor.schema.js, you can just generate this file. (my script for generating json from TargetDescriptor.schema.js https://github.com/lukaw3d/parcel-bundler-json-schemas/blob/main/main.js)